metapool/plate.py

```diff
@@ -0,0 +1,192 @@
+import click
+import pandas as pd
+
+from math import ceil
+from datetime import datetime
+
+
+class Message(object):
+    def __init__(self, message):
```

Comment:
if this _color is private, does it need a method to be set?

Reply:
Well, I think color is really tied to the class, so I think it should really be a class attribute. Fixing.